src/events/_event-post-macros.html

```diff
@@ -353,3 +354,135 @@ <h2 class="event-venue_heading">{{ city }}</h2>
   </div>
   {% endif %}
 {% endmacro %}
+
+{# ==========================================================================
+
+   event_mailto_button()
+
+   ==========================================================================
+
+   Description:
+
+   Create an event mailto button when given:
+
+   event: An event from a query result.
+
+   ========================================================================== #}
+
+{% macro event_mailto_button( event ) %}
+{% if event.rsvp %}
+    {% set rsvp = {
+          'contact': event.rsvp.contact | list | last,
+          'subject': event.rsvp.email_subject,
+          'body'   : event.rsvp.email_body | striptags
+      }
+    %}
+{% else %}
+    {% set rsvp = {
+          'contact': 'reserve@cfpb.gov',
+          'subject': 'Event RSVP',
+          'body'   : 'To RSVP, please fill in your first and last
+                     name below and send this email.%0D%0A%0D%0A
+                     First name:%0D%0A
+                     Last name:'
+      }
+    %}
+{% endif %}
+<a href="mailto:{{rsvp.contact}}?subject={{rsvp.subject}}&body={{rsvp.body}}"
+   class="btn">
+   <span class="btn_icon__left cf-icon cf-icon-email-social"></span>
+      {{ rsvp.contact | safe }}
+</a>
+{% endmacro %}
+
+
+{# ==========================================================================
+
+   event_media()
+
+   ==========================================================================
+
+   Description:
+
+   Create event archive media markup when given:
+
+   event: An event from a query result.
+
+   TODO: Replace lorem ipsum and placeholder image.
+
+   ========================================================================== #}
+
+{% macro event_media( event ) %}
+{% set col_classes = 'content-l_col
+                      content-l_col-1-2
+                      block
+                      block__padded-top
+                      block__flush-top
+                      block__flush-bottom'
+%}
+
+<div class="block block__border-top">
+    <div class="event-archive_media-container
+                content-l
+                content-l__large-gutters">
+        <section class="{{ col_classes }} block__padded-bottom">
+            <h4>Photography</h4>
+            <p class="u-flexible-container">
+                <img class="u-flexible-container_inner"
+                     src='https://placeholdit.imgix.net/~text?&w=360&h=202'>
+            </p>
+            <p class="speech_link-container">
+                <a href="#" class="jump-link
```

Comment:
Minor: Remove `href="#"` while `u-link__disabled` is present. Having the href means the link can still be clicked.